target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-rb3011uias.dts

```diff
@@ -0,0 +1,599 @@
+#include "qcom-ipq8064-v2.0.dtsi"
```

Comment:
Missing SPDX license identifier.

Reply:
Not my code to add a license. @adron-s, what license do you want to use?